sgkit/io/vcf/__init__.py

```diff
@@ -0,0 +1,17 @@
+try:
+    from .vcf_partition import partition_into_regions  # noqa: F401
+    from .vcf_reader import vcf_to_zarr, vcf_to_zarrs, zarrs_to_dataset  # noqa: F401
+
+    __all__ = [
+        "partition_into_regions",
+        "vcf_to_zarr",
+        "vcf_to_zarrs",
+        "zarrs_to_dataset",
+    ]
+except ImportError as e:
+    msg = (
```

Comment:
I wonder if we should also check here if the OS is Windows, and if it is, instead of suggesting `pip install` straight up say that `sgkit-vcf` doesn't support Windows.

Reply:
I agree, it will save save people some time if we just tell them sgkit-vcf isn't supported on windows.

Reply:
And we could also mentioned that if the user wants to work with VCF data on Windows, they could convert that data into zarr and then still use sgkit on Windows via `read_vcfzarr`, essentially to prevent the impression of sgkit+vcf on Windows being a complete showstopper.

Reply:
Good suggestion. I've added a Windows-specific message, referring users to the documentation, which is where I'll add something about the `read_vcfzarr` workaround.